BookingQueueSubscriber/BookingQueueSubscriber.Services/NotificationApi/NotificationService.cs

```diff
@@ -1,32 +1,132 @@
+        /// <param name="password"></param>
+        /// <returns></returns>
+        Task SendNewUserSingleDayHearingConfirmationEmail(HearingDto hearing, ParticipantDto participant, string password);
+        
+        /// <summary>
+        /// Send a single day hearing confirmation to existing users. Part of the 2nd of 3 new template
+        /// </summary>
+        /// <param name="hearing"></param>
+        /// <param name="participant"></param>
+        /// <returns></returns>
+        Task SendExistingUserSingleDayHearingConfirmationEmail(HearingDto hearing, ParticipantDto participant);
+
+        /// <summary>
+        /// Send a multi-day hearing confirmation to new and existing users. Part of the 2nd of 3 new template
+        /// </summary>
+        /// <param name="hearing"></param>
+        /// <param name="participant"></param>
+        /// <param name="days"></param>
+        /// <returns></returns>
         Task SendMultiDayHearingNotificationAsync(HearingDto hearing, IList<ParticipantDto> participants, int days);
+        
+        
+        
+        
+        
+        
+        
+        
+        // Task SendNewUserAccountNotificationAsync(Guid hearingId, ParticipantDto participant, string password);
+        // Task SendNewHearingNotification(HearingDto hearing, IEnumerable<ParticipantDto> participants);
+        // Task SendHearingAmendmentNotificationAsync(HearingDto hearing, DateTime originalDateTime, IList<ParticipantDto> participants);
+        // Task SendNewUserWelcomeEmail(HearingDto hearing, ParticipantDto participant);
+        // Task SendNewUserAccountDetailsEmail(HearingDto hearing, ParticipantDto participant, string userPassword);
+        // 
+        //
+        
+        // Task SendExistingUserAccountDetailsEmail(HearingDto hearing, ParticipantDto participant);
     }
 
     public class NotificationService : INotificationService
     {
         private readonly INotificationApiClient _notificationApiClient;
         private readonly IBookingsApiClient _bookingsApiClient;
         private readonly ILogger<NotificationService> _logger;
+        private readonly IFeatureToggles _featureToggles;
+        private readonly IUserService _userService;
 
-        public NotificationService(INotificationApiClient notificationApiClient, IBookingsApiClient bookingsApiClient, ILogger<NotificationService> logger)
+        public NotificationService(INotificationApiClient notificationApiClient, IBookingsApiClient bookingsApiClient, ILogger<NotificationService> logger, IFeatureToggles featureToggles, IUserService userService)
         {
             _notificationApiClient = notificationApiClient;
             _bookingsApiClient = bookingsApiClient;
             _logger = logger;
+            _featureToggles = featureToggles;
+            _userService = userService;
         }
 
+        // public async Task SendNewUserAccountNotificationAsync(Guid hearingId, ParticipantDto participant, string password)
```

Comment:
could we remove this commented out code?

Reply:
done